### PR TITLE
Use Semantic UI for select / collection_select

### PR DESCRIPTION
**Pull request by smangelsdorf**

### CODE REVIEW

File: lib/lipstick/helpers/semantic_form_builder.rb [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
 class SemanticFormBuilder < ActionView::Helpers::FormBuilder
+  attr_reader :template
+  delegate :content_tag, :tag, :concat, :icon_tag, to: :template
+
   def date_field(field, **opts)
     opts[:class] = "#{opts[:class]} date-picker".strip
     text_field(field, opts)
   end
+
+  def select(method, choices = nil, opts = {}, html_opts = {}, &block)
```
Comment (rianniello) [inline lib/lipstick/helpers/semantic_form_builder.rb]: ```select``` can be made private?

Comment (smangelsdorf) [inline lib/lipstick/helpers/semantic_form_builder.rb]: [It's part of the Rails API](http://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-select)

Comment (rianniello) [inline lib/lipstick/helpers/semantic_form_builder.rb]: If this is the case, does it make sense to have a unit test specifically for it? 

I can see you've covered collection_select but not select.

Comment (smangelsdorf) [inline lib/lipstick/helpers/semantic_form_builder.rb]: You win this round @rianniello

Comment (rianniello) [inline lib/lipstick/helpers/semantic_form_builder.rb]: All looks good to me.

**Pull request merged**